tests/base/model_optimizers.py

```diff
@@ -34,6 +34,14 @@ def configure_optimizers__multiple_optimizers(self):
         optimizer2 = optim.Adam(self.parameters(), lr=self.learning_rate)
         return optimizer1, optimizer2
 
+    def configure_optimizers__multiple_optimizers_frequency(self):
```

Comment:
```suggestion
    def configure_optimizers___multiple_optimizers_frequency(self):
```
pls check the other syntax, but I gues it shall be 3x "_"

Reply:
it's 2 everywhere.